src/v/redpanda/CMakeLists.txt

```diff
@@ -11,7 +11,9 @@ set(swags
   transaction
   cluster
   debug
-  shadow_indexing)
+  usage
+  shadow_indexing
+  redpanda_services_restart)
```

Comment:
These two are backported as well but they are not the part of the original PR, is this intended?

Reply:
i just edited that, the conflict was there

Reply:
https://github.com/redpanda-data/redpanda/blob/v23.1.x/src/v/redpanda/CMakeLists.txt#L14